src/core/var/rsingleTag.js

```diff
@@ -1,5 +1,5 @@
 define( function() {
 
 	// Match a standalone tag
-	return ( /^<([\w-]+)\s*\/?>(?:<\/\1>|)$/ );
+	return ( /^<([a-zA-Z][^\s\/\0>]*)\s*\/?>(?:<\/\1>|)$/ );
```

Comment:
It's only used in association with `jQuery.parseHTML`; any gains would be unnoticeable in the cost of creating elements.